include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -3053,137 +2866,108 @@ namespace internal
+                                                       eval.begin_hessians(),
+                                                       scratch_data,
+                                                       subface_index);
       else
-        FEFaceEvaluationImpl<
-          false,
-          dim,
-          fe_degree,
-          n_q_points_1d_actual,
-          VectorizedArrayType>::evaluate_in_face(n_components,
-                                                 data,
-                                                 temp1,
-                                                 values_quad,
-                                                 gradients_quad,
-                                                 hessians_quad,
-                                                 scratch_data + 3 *
-                                                                  n_components *
-                                                                  dofs_per_face,
-                                                 evaluate_values,
-                                                 evaluate_gradients,
-                                                 evaluate_hessians,
-                                                 subface_index);
+        FEFaceEvaluationImpl<false,
+                             dim,
+                             fe_degree,
+                             n_q_points_1d_actual,
+                             Number>::evaluate_in_face(n_components,
+                                                       evaluation_flag,
+                                                       shape_data,
+                                                       temp,
+                                                       eval.begin_values(),
+                                                       eval.begin_gradients(),
+                                                       eval.begin_hessians(),
+                                                       scratch_data,
+                                                       subface_index);
 
       if (face_orientation)
-        adjust_for_face_orientation(dim,
-                                    n_components,
-                                    face_orientation,
-                                    orientation_map,
-                                    false,
-                                    evaluate_values,
-                                    evaluate_gradients,
-                                    evaluate_hessians,
-                                    data.n_q_points_face,
-                                    scratch_data,
-                                    values_quad,
-                                    gradients_quad,
-                                    hessians_quad);
+        adjust_for_face_orientation(
+          dim,
+          n_components,
+          evaluation_flag,
+          &eval.get_orientation_map()(face_orientation, 0),
```

Comment:
Not sure if this will work in the context of ECL.

Reply:
Exactly this is the place where we could extend things by a function similar to the one I use in `gather_evaluate`.

Reply:
See https://github.com/dealii/dealii/pull/13063^^